ilc/server/tailor/fetch-template.js

```diff
@@ -15,7 +17,19 @@ module.exports = (templatesPath, router, configsInjector, newrelic, registryServ
     request,
     parseTemplate
 ) => {
-    const registryConfig = await registryService.getConfig();
+    let registryConfig = await registryService.getConfig();
+
+    const { cookie } = request.headers;
```

Comment:
Let's expect cookie value to be URL encoded. To mitigate risk of having issues because of the special characters inside JSON.